Solana/src/test/java/com/solana/models/MetaplexInfra.kt

```diff
@@ -0,0 +1,18 @@
+package com.solana.models
+
+import com.solana.core.PublicKey
+import com.solana.vendor.borshj.BorshCodable
+import com.solana.vendor.borshj.FieldOrder
+
+data class MetaplexMeta(
+    @FieldOrder(0) val key: Byte,
```

Comment:
The number in the order annotation is even sorta arbitrary, all that matters is that the numbers are from low to high alongside the relevant field.